### PR TITLE
Provisioning of the ETCD cluster

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/KAC-173

SEE: https://github.com/keboola/keboola-as-code/pull/770

Changes:
- Added provisioning of ETCD cluster (`replicas=1`) for Templates API.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,11 @@ COPY Makefile /tmp/build/Makefile
 COPY scripts  /tmp/build/scripts
 RUN cd /tmp/build && make tools && rm -rf /tmp/build
 
+# Install envsubstr and helm
+RUN curl -L https://github.com/a8m/envsubst/releases/download/v1.2.0/envsubst-$(uname -s)-$(uname -m) -o /usr/local/bin/envsubst && \
+    chmod +x /usr/local/bin/envsubst && \
+    curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+
```
Comment (webrouse) [inline Dockerfile:24]: Pridane do DEV image (pre vyvoj na MacOS).

File: provisioning/azure/scripts/deploy.sh [hunk 2]
```diff
@@ -17,16 +17,26 @@ kubectl delete all -n default -l app=templates-api --ignore-not-found
...
+while [[ -z "$TEMPLATES_API_IP" && $TIME_WAITED -lt 15*60 ]]; do
+    echo "Waiting for Templates API ingress IP..."
+    sleep 10;
+    TIME_WAITED=$((TIME_WAITED + 10))
+    TEMPLATES_API_IP=$(kubectl get services \
+      --selector "app=templates-api" \
+      --namespace templates-api \
+      --no-headers \
+      --output jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")
+done
```
Comment (webrouse) [inline provisioning/azure/scripts/deploy.sh:39]: Neviem teda, ci bol problem aj v tomto alebo nie, ale zostat to tu moze:
https://github.com/keboola/keboola-as-code/pull/770#issuecomment-1225872234

Comment (ondrejhlavacek) [reply provisioning/azure/scripts/deploy.sh:39]: to se hodí, dal jsem to už do #778

**Comment by webrouse** [comment]
(rebased)

**Comment by webrouse** [comment]
(rebased)

**Review by ondrejhlavacek — APPROVED** [review]
lgtm, díky!

**Pull request merged**